python/snewpy/models/ccsn.py

```diff
@@ -242,6 +243,9 @@ def __init__(self, base=None, mass=None, eos='LS220', *, progenitor_mass=None):
             return super().__init__(eos=eos, progenitor_mass=progenitor_mass)
 
 OConnor_2013.__init__.__doc__=dedent(OConnor_2013.__init__.__doc__)+_OConnor_2013_new.__init__.__doc__
+#make sure that only the latest class is present in the models table
```

Comment:
why you only do this for OConnor_2013? I didn't follow that

Reply:
Because `OConnor_2013` is slightly special: all the classes are automatically recorded in `all_models` when they are created in `@RegistryModel` decorator. 

But `OConnor_2013` inherits from such `@RegistryModel` class, so in the `all_models` there is a parent of `OConnor_2013`. So, first we remove this parent, and then add this child class.

Reply:
I forgot to mention: this kind of "workarounds" will not be needed and will be removed when we get to the v2.0 and cleanup the interfaces.

Reply:
okey I see, thanks for explaining